docs/v2-system-addon/data_dictionary.md

```diff
@@ -170,7 +171,8 @@ and losing focus. | :one:
 | `ua` | [Auto populated by Onyx] The user agent string. | :two:
 | `unload_reason` | [Required] The reason the Activity Stream page lost focus. | :one:
 | `url` | [Optional] The URL of the recommendation shown in one of the highlights spots, if any. | :one:
-| `value` | [Required] An integer that represents the measured performance value. Can store counts, times in milliseconds, and should always be a positive integer.| :one:
+| `value` (performance) | [Required] An integer that represents the measured performance value. Can store counts, times in milliseconds, and should always be a positive integer.| :one:
+| `value` (event) | Has {"icon_type", "card_type"}, when user action was triggered from a topsite, or {"card_type"} otherwise | :one:
```

Comment:
How about `... | [Optional] An object with keys "icon_type" and "card_type" to record the extra information of event ping |`?